prr/services/providers/elevenlabs/tts.py

```diff
@@ -0,0 +1,27 @@
+from elevenlabs import generate, set_api_key
+
+from prr.services.service_base import ServiceBase
+from prr.utils.config import load_config
+from prr.utils.response import ServiceResponse
+
+config = load_config()
+set_api_key(config.get("ELEVENLABS_API_KEY", None))
+
+
+# Eleven Labs model provider class
+# models: "eleven_monolingual_v1", "eleven_multilingual_v1"
+class ServiceElevenLabsTTS(ServiceBase):
+    provider = "elevenlabs"
+    service = "tts"
+    options = ["voice"]
+
+    def run(self):
+        audio = generate(
+            text=self.request.prompt_content,
+            voice=self.option("voice"),
+            model=self.service_config.model_name(),
+        )
+
+        self.response = ServiceResponse(audio)
```

Comment:
right now what we have is only the length of response, without actually storing/showing the response.

```python
import os
path = "./audio.mp3"
f = open(path, 'wb)
f.write(audio)
```
would also save the audio result to the mp3 file

Reply:
The file gets saved with using -l option as well as you can use -q to pipe it to mp3 file.